src/QirTools/pyqir/qirlib/build.rs

```diff
@@ -0,0 +1,30 @@
+use std::env;
+use std::error::Error;
+use std::io::{self, Write};
+use std::path::PathBuf;
+use std::process::Command;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    println!("cargo:rerun-if-env-changed=TARGET");
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-changed=runtime.csproj");
+
+    let root_path = env::var("CARGO_MANIFEST_DIR")?;
+    let packages = PathBuf::from(format!("{}/packages", root_path.clone()));
+
+    let output = Command::new("dotnet")
```

Comment:
It looks like this is used to make sure the QIR runtime is made available by the relevant NuGet package, is that correct? If so, a comment indicating such may be helpful here.